presto-cassandra/src/main/java/com/facebook/presto/cassandra/CassandraTableLayoutHandle.java

```diff
@@ -28,22 +28,24 @@
 {
     private final CassandraTableHandle table;
```

Comment:
This class is really error pron. The serialization roundtrip must yield the same object. It is not true for this class. Please refactor it. It shouldn't be hard. And add json serialization round trip unit test.